x-pack/filebeat/tests/integration/otel_test.go

```diff
@@ -44,11 +43,7 @@ output:
     password: testing
     index: %s
 queue.mem.flush.timeout: 0s
-processors:
-    - add_host_metadata: ~
-    - add_cloud_metadata: ~
-    - add_docker_metadata: ~
-    - add_kubernetes_metadata: ~
```

Comment:
I think we still need this here. Beats receivers have these processors loaded by default, but since we use this config for normal beats as well they do not load them by default, which results in differences in the ingested docs.